compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -299,79 +295,24 @@ fn add_unused_functions<'ll, 'tcx>(cx: &CodegenCx<'ll, 'tcx>) {
 
     let codegenned_def_ids = tcx.codegened_and_inlined_items(());
 
-    let mut unused_def_ids_by_file: FxHashMap<Symbol, Vec<DefId>> = FxHashMap::default();
     for &non_codegenned_def_id in all_def_ids.difference(codegenned_def_ids) {
```

Comment:
Pre-existing, but consider renaming `all_def_ids` to something like `eligible_def_ids` since we remove some of them in the filter_map.

Actually, we do more filtering below and I'm not sure I understand why it's split up like that. It seems like we're doing a lookup on the def_id in both cases.